wagtail/images/views/chooser.py

```diff
@@ -156,6 +158,45 @@ def image_chosen(request, image_id):
     )
 
 
+def duplicate_found(request, new_image, duplicates):
+    next_step_url = (
+        "wagtailimages:chooser_select_format"
+        if request.GET.get("select_format")
+        else "wagtailimages:image_chosen"
+    )
+
+    confirm_duplicate_upload_action = reverse(next_step_url, args=(new_image.id,))
+
+    existing_image = duplicates[0]
```

Comment:
`.first()` may be more appropriate here to avoid loading the queryset into memory

Reply:
I've updated it, thanks!